api-list.md

```diff
@@ -15,20 +34,19 @@
 ```
 API Root Path  
 |-- users
+|-- nodes (limited api)
 |-- organizations
 |   |-- nodes
 |   |-- occasions
 |   |-- containers
 |   |-- surveys
 |   |   |-- node
-|-- offerings
-|   |-- survey
-|-- registrations
-|   |-- node
-|   |-- offering
-|   |-- user
-|   |-- offline
-|-- payments
```

Comment:
I think payments should only be accessible in two contexts:
* by organization
* by a particular user

so we could have `/organizations/:org_id/payments` and `/users/:my_id/payments`

Reply:
should probably also have an api route `/users/myself` that is an alias for `/users/<my ID>`

Reply:
Good call on both counts.